Calculator/CalculatorBrain.swift

```diff
@@ -18,14 +21,61 @@ class CalculatorBrain {
     var operatorType: OperatorType?
     
     func addOperandDigit(_ digit: String) -> String {
-        
+        guard operatorType == nil else {
+            operand2String.append(digit)
+            return operand2String
+        }
+        operand1String.append(digit)
+        return operand1String
     }
     
     func setOperator(_ operatorString: String) {
-        
+
+        switch operatorString {
+        case OperatorType.addition.rawValue:
+            operatorType = .addition
+        case OperatorType.subtraction.rawValue:
+            operatorType = .subtraction
+        case OperatorType.multiplication.rawValue:
+        operatorType = .multiplication
+        case OperatorType.division.rawValue:
+            operatorType = .division
+        default:
+            print("Error setting operator")
+        }
```

Comment:
I personally am not a big fan of switch statements. They can be easy to read but they are a lot of work. And here a switch statement is not needed because it is just a string that has countless possible cases. 

You could have directly set the value of the operatorType instead of setting up possible scenarios for what the operatorType should do depending on the operatorString. 

So something like: 
```
func setOperator(_ operatorString: String) {
        if let thisOperatorType = OperatorType(rawValue: operatorString) {
            operatorType = thisOperatorType
        }
    }
```